test/run-integration.js

```diff
@@ -112,6 +112,12 @@ function initPlaces(placesAppId, placesApiKey) {
     }, function(e) {
       t.fail(e);
     });
+
+    places.reverse('48.880397, 2.326991').then(function(res) {
+      t.ok(res.nbHits, 'We got some results by querying `48.880397, 2.326991`');
```

Comment:
shouldn't this be > 0 or something?

Reply:
`nbHits` is `0` when no hits – the intent would indeed be clearer with the expression `nbHits > 0`.